### PR TITLE
Add features 5-7.

### PR DESCRIPTION
**Pull request by github-workshop-demo**
I did a lot of work here. Refactored a thing. Fixed a bug. And released features 5-7.

Here is the new feature:

image

jira.com/1234

### CODE REVIEW

File: 7.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+asdf7
```
Comment (AndrewSouthpaw) [inline 7.txt]: There's a bug here, this should say `defg7`

File: 7a.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+asdf7a
```
Comment (AndrewSouthpaw) [inline 7a.txt:1]: This looks really great! 👍🏼

**Review by AndrewSouthpaw — CHANGES_REQUESTED** [review]
Overall looks good, just a few changes.

File: 7.txt (hunk 1 above)
Comment (github-workshop-demo) [reply 7.txt]: Made a fix, take a look.

**Review by AndrewSouthpaw — APPROVED** [review]
LGTM. 🚢

**Pull request merged**